src/domain/entities/TrackedEntity.ts

```diff
@@ -0,0 +1,97 @@
+import { Id } from "./Base";
+
+export interface TrackedEntity {
+    trackedEntity: Id;
+    trackedEntityType: Id;
+    createdAt: ISODate;
+    createdAtClient: ISODate;
+    updatedAt: ISODate;
+    updatedAtClient: ISODate;
+    orgUnit: Id;
+    inactive: boolean;
+    deleted: boolean;
+    potentialDuplicate: boolean;
+    relationships: never[];
+    attributes: Attribute[];
+    enrollments: Enrollment[];
+    programOwners: ProgramOwner[];
+}
```

Comment:
[comment clean arch] Typically our entities have "nicer" properties names/values (for example, the first could be simply: `id: Id`, and others Refs). But I understand the closer the API is to the domain, the fewer conversions back-and-forth we need, so no need to change it for so little gain!

Reply:
I'll keep it in mind for the next one 🙂